### PR TITLE
feat(TG-379): Navigate to constructor on create click

### PR DESCRIPTION
**Pull request by valakar**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/garage-it/SmartHouse-frontend/pull/163" target="_blank">ReviewNinja

File: src/app/constructor/constructor.component.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { Component, OnInit } from '@angular/core';
```
Comment (velmyk) [inline src/app/constructor/constructor.component.ts]: You do not use OnInit here

Comment (borys-ihnatyev) [inline src/app/constructor/constructor.component.ts]: !fixed

File: src/app/constructor/constructor.template.html [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<sh-map-constructor></sh-map-constructor>
+
+<div>
+    <button [routerLink]="['../']">
```
Comment (Hliebov) [inline src/app/constructor/constructor.template.html:4]: Doesn't it  work with '../'?

**Pull request merged**